### ISSUE TITLE
Add it to the official deno module list?

### ISSUE THREAD

**Issue by DzhideX** [NMathar/deno-express#14]
The official deno third party module list can be found at this link: https://deno.land/x

I'm not sure if I'm missing something though!

**Comment by NMathar** [comment]
Yeah i am thinking of it since I startet this project and  I never thought that my module would grow this fast 😅 I don’t have that much time at the moment to dive in to things but I hold this issue as a todo.
